saltstack/final/salt/metadata/tmp/generate-tags.sh

```diff
@@ -13,6 +13,10 @@ if [[ "$OS" == "redhat8" ]]; then
     TAGS=$(echo $TAGS | jq --arg fipsmode $FIPSMODE -r '. + {"fips-mode": $fipsmode}')
 fi
 
+if [ "${CLOUD_PROVIDER}" != "AWS_GOV" ]; then
```

Comment:
As part of CB-22247 we've already enabled CIS for Gov Cloud images as well - this "if" is no longer needed.